examples/configs/autoalign/autoalign_factcheck_config/config.yml

```diff
@@ -1,12 +1,19 @@
 models:
   - type: main
     engine: openai
-    model: gpt-3.5-turbo
+    model: gpt-3.5-turbo-instruct
```

Comment:
I think gpt-3.5-turbo is more powerful? any particular reason for this change?

Reply:
I saw that in their RAG flow, they are using this model. Maybe it is due to its instruction-following capability

Reply:
okay, then it's good to keep